Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -34,7 +37,76 @@ public static class TextTable
         /// </summary>
         public static string Build(string message, int padding)
         {
-            return "";
+            if (string.IsNullOrEmpty(message))
+            {
+                return "";
+            }
+            // make a array so I can find the longest word. 
+
+            var arrayOfLines = message.Split(Environment.NewLine);
+
+            // Find length of longest word 
+            var lengthLongestWord = GetLongestWordLength(arrayOfLines);
+
+            // print table 
+
+            var table = new StringBuilder();
+            AddTopOrBottomLineToTable(padding, lengthLongestWord, table);
+            AddEmptyLinesToTable(padding, lengthLongestWord, table);
+            AddTextToTable(padding, arrayOfLines, lengthLongestWord, table);
+            AddEmptyLinesToTable(padding, lengthLongestWord, table);
+            AddTopOrBottomLineToTable(padding, lengthLongestWord, table);
+            return table.ToString();
+
+        }
+
+        private static void AddTextToTable(int padding, string[] arrayOfLines, int lengthLongestWord, StringBuilder table)
+        {
+            foreach (var word in arrayOfLines)
+            {
+                table.Append("|");
+                table.Append(String.Empty.PadRight(padding, ' '));
+                table.Append(word);
+                table.Append(String.Empty.PadRight(lengthLongestWord - word.Length + padding, ' '));
+                table.Append("|");
+                table.Append(Environment.NewLine);
+
```

Comment:
Spacing

Reply:
What do you exactly mean with spacing.  According to the editor there are on the right place or do you mean the empty lines under it

Reply:
Empty line. You have too many empty lines (2 in a row sometimes)

Reply:
oke, then that is solved